language/stdlib/modules/libra_block.move

```diff
@@ -55,7 +56,14 @@ module LibraBlock {
 
       // Currently distribute once per-block.
       // TODO: Once we have a better on-chain representation of epochs we will make this per-epoch.
-      TransactionFee::distribute_transaction_fees<LBR::T>();
+        TransactionFee::distribute_transaction_fees<LBR::T>();
```

Comment:
is there an odd spacing here?

Reply:
True, in fact, the original file has incorrect 2-space ident, should be 4-space ident. Will fix that.